### PR TITLE
Display badge label on dataset and org page

### PR DESCRIPTION
**Pull request by noirbizarre**

### CODE REVIEW

File: udata/templates/dataset/display.html [hunk 1]
```diff
@@ -190,7 +190,7 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
                                 </a>
                                 {% for badge in dataset.badges %}
                                     <a href="{{ url_for('front.search', badge=badge) }}">
-                                        {{ badge }}</a>{% if not loop.last %}, {% endif %}
+                                        {{ dataset.__badges__[badge.kind] }}</a>{% if not loop.last %}, {% endif %}
```
Comment (davidbgk) [inline udata/templates/dataset/display.html]: Do we want to expose `__private__` in templates? Why iterating over badges is not enough?

Comment (noirbizarre) [inline udata/templates/dataset/display.html]: Because the way it's done, right now badge object doesn't has access to the label :/

Comment (davidbgk) [inline udata/templates/dataset/display.html]: Oh I get it, too bad :(

**Comment by noirbizarre** [comment]
Changed to hide the private `__badges__` access.

File: udata/templates/organization/display_base.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
                                 <a href="{{ url_for('front.search', badge=badge) }}"
                                     title="{{ _('See all organizations with that badge.') }}">
                                     <span class="glyphicon glyphicon-bookmark"></span>
-                                    {{ badge }}</a>
+                                    {{ org.badge_laebl(badge) }}</a>
```
Comment (davidbgk) [inline udata/templates/organization/display_base.html]: Typo

**Comment by davidbgk** [comment]
I really prefer that way, thanks!

**Pull request merged**